ESP8266/README.md

```diff
@@ -1 +1,45 @@
 
+# Welcome to IoT-Spot👋
```

Comment:
Delete this README from the `ESP8266/README.md` This got copied from the main README of the project

Reply:
I followed the main readme file but i have made the necessary changes for the ESP8266
should I delete the whole file? 
@Harshil-Jani